Sources/CodeGen/CXX/CXXTranspiler.swift

```diff
@@ -603,4 +600,63 @@ public struct CXXTranspiler {
 
   }
 
+  // MARK: entry point function body
+
+  /// Returns the transpiled body of the entry point function in `source`.
+  func entryPointBody(module source: ModuleDecl.Typed) -> CXXScopedBlock? {
+    let elements = Array(source.topLevelDecls)
+    return elements.first(transformedBy: { (d) in
+      entryPointBody(topLevel: d, in: source)
+    })
```

Comment:
```suggestion
    source.topLevelDecls.first(transformedBy: { (d) in
      entryPointBody(topLevel: d, in: source)
    })
```
One of the big advantage of extending `Sequence` is that you shouldn't have to transform other sequences/collections to `Array` in order to apply generic algorithms.

Reply:
sorry. this was a leftover from when I was trying to debug why `first(transformedBy:)` doesn't properly work. Fixed